lib/commands/base.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+function BaseCommand(server) {
+  this.server = server;
+}
+
+BaseCommand.prototype.getDbAndCollectionName = function(clientReqMsg) {
+   var fullCollectionName = clientReqMsg.fullCollectionName;
+   var firstDot = fullCollectionName.indexOf('.');
```

Comment:
I wish you had lodash :(

http://jsperf.com/lodash-contains-vs-native-indexof/3